infrastructure/docker/build/docker-compose.yml

```diff
@@ -42,6 +42,14 @@ services:
     volumes:
       - ../../..:/trafficcontrol:z
 
+  traffic_ops_ort_build:
+    image: traffic_ops_ort_builder
+    build:
+      dockerfile: infrastructure/docker/build/Dockerfile-traffic_ops_ort
+      context: ../../..
```

Comment:
Since we are hosting the builder images on Docker Hub now, the image field should reference the `master` tag of an image named `trafficcontrol/traffic_ops_ort_builder`, which also needs to be created. The build section can stay for testing but should be commented out.

Creating the Docker Hub repo for this image can be super easy because there is no need to enter any build rules, as that would involve linking it to a GitHub repo and unlinking the GH repo would delete all of the existing build rules anyway.